src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -188,7 +188,7 @@
 	private final VComboBox<String> mobModelTexture = new SearchableComboBox<>();
 	private final VComboBox<String> mobModelGlowTexture = new SearchableComboBox<>();
 
-	private final JEntityDataList entityDataList = new JEntityDataList(mcreator);
+	private final JEntityDataList entityDataList = new JEntityDataList(mcreator, this);

```

Comment:
This will not build because it takes the GUI instance that is not existing by this moment. Move the assignment to `initGUI()` method.